device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -1246,30 +1251,22 @@ private void reconnect(TransportException transportException)
 
         // reconnection may have failed, so check last retry decision, check for timeout, and check if last exception
         // was terminal
-        try
```

Comment:
This section is a bit hard to see the diff on, but basically the try/catch was removed since the caught exception is no longer thrown